xmtp-inbox-ios/Views/NewConversationView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  NewConversationView.swift
+//  xmtp-inbox-ios
+//
+//  Created by Elise Alix on 2/10/23.
+//
+
+import AlertToast
+import Combine
+import SwiftUI
+import XMTP
+
+struct NewConversationView: View {
+	let client: XMTP.Client
+	var onCreate: (DB.Conversation) -> Void
+	let searchTextPublisher = PassthroughSubject<String, Never>()
+
+	@Environment(\.dismiss) var dismiss
+	@FocusState var isFocused
+	@State private var searchText = ""
+	@State private var searchResults = [String]()
+	@State private var error: String?
+
+	var body: some View {
+		NavigationView {
+			ZStack {
+				Color.backgroundPrimary.edgesIgnoringSafeArea(.all)
+				List {
+					TextField("new-message-prompt", text: $searchText)
+						.focused($isFocused)
+						.onAppear {
+							self.isFocused = true
+						}
+						.onChange(of: searchText) { searchText in
+							searchTextPublisher.send(searchText)
+						}
```

Comment:
Could be nice to extract an `AutoFocusedTextField` component since I feel like we'll want this in more places.